orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/pipeline/servergroup/DestroyServerGroupStage.groovy

```diff
@@ -40,10 +40,14 @@ class DestroyServerGroupStage extends TargetServerGroupLinearStageSupport implem
   }
 
   private static void addDisableStage(Map<String, Object> context, StageGraphBuilderImpl graph) {
-    graph.add {
-      it.name = "disableServerGroup"
-      it.type = getType(DisableServerGroupStage)
-      it.context.putAll(context)
+    boolean skipDisable = (boolean)context.getOrDefault("forceSkipDisable", false)
```

Comment:
nit ... maybe `shouldDisableBeforeDestroy` (default to `true`) vs `forceSkipDisable` (default to `false`).

Otherwise I think this is a useful flag!

Reply:
I prefer props that aren't there to be `false`. But in this very limited (from code perspective) use case it feels less consequential. I also thought having more strong language e.g. `force` would be good since most people shouldn't be setting the flag so when it's set, hopefully it stands out a bit

Reply:
Agree with the default-to-false idea, makes sense to me.